AFQ/tests/test_definitions.py

```diff
@@ -23,6 +24,15 @@ def test_str_instantiates_mixin():
                 thresh_image_from_str.upper_bound)
 
 
+def test_name_from_path():
+    path = "/data/sub-01/ses-01/dwi/sub-01_ses-01_desc-icvf_dwi.nii.gz"
```

Comment:
Actually this would have failed before the change, right?

Reply:
Yes!